README.md

```diff
@@ -1,30 +1,68 @@
 # PUN-street-access
+
 PUA
 
+# how to run it
 
-# how to run it 
 ## frontend
+
 ### env
+
 ```
 cd frontend
 npm install
 ```
 
 ### run
+
 ```bash
 cd frontend
 npm run dev -- --open
 ```
+
 it will auto reload after code change
 
 ## backend
 
 ### env
 
 not yet use
+
 ```bash
+
 ```
+
 ### run
+
 ```bash
 go run .
 ```
+
+## sql
+
+### start
+
+> detach is background run, if want see its output remove --detach
+
+```bash
+cd sql
+docker-compose up --detach
+```
+
+### into postgres
+
+```bash
+psql -h localhost -p 5432 -U user
```

Comment:
use `docker exec -it <container> psql ...` so users don't have to install postgres on their machine

Reply:
listen good to can add new line to readme

Reply:
fixed